src/tests/api/AbortApi.test.ts

```diff
@@ -15,13 +15,13 @@ describe("E2E Happy Path /abort enpoint", () => {
 	it("E2E Happy Path Journey - Abort Session", async () => {
 		expect(sessionId).toBeTruthy();
 		const response = await postAbortSession(abortPayload, sessionId);
-		expect(response.request.path).toContain("/redirect?error=access_denied&state=");
+		expect(response.headers.location).toContain("/redirect?error=access_denied&state=");
```

Comment:
It's strange to me that this isn't case sensitive, but I guess if it works it works 🤷‍♀️